tests/xx-epub-template/OPS/package.opf

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
   <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
-    <dc:title>Title</dc:title>
-    <dc:creator>Creator</dc:creator>
-    <dc:date>2021-01-01</dc:date>
-    <dc:identifier id="pub-id">NOID</dc:identifier>
+    <dc:identifier id="pub-id">TODO: spec-anchor</dc:identifier>
+    <dc:title>TODO: Concise Test Title</dc:title>
+    <dc:description>
+        TODO: A longer description of the test for the generated test report.
+    </dc:description>
+    <dc:coverage>TODO: Report category</dc:coverage>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-33/#spec-anchor</meta>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-rs-33/#spec-anchor</meta>
+
+    <dc:creator>W3C EPUB 3 Working Group</dc:creator>
```

Comment:
How do you feel about this creator vs putting the respective test author's name in the file?

Reply:
I am, mildly, in favour of using the author's name.

Reply:
Done.